andela/hc-shleemypants
# #155965542 allocate check owners 155965542

#### What does this PR do ?
Allocate different stakeholders as owners of different HealthCheck

#### Description of tasks to be completed
Team access should not guarantee all existing individual healthchecks jobs access. A user granting team access to their account should be able to assign different jobs to the person they are granting access to

How should this be manually tested?

check out project code:

  $ git clone https://github.com/healthchecks/healthchecks.git
install requirements (Django, ...) into virtualenv:

  $ pip install -r healthchecks/requirements.txt
What are the relevant pivotal tracker stories?

155965542
